.github/workflows/lint.yml

```diff
@@ -0,0 +1,35 @@
+name: Lint
+
+on:
+  # Trigger the workflow on push or pull request,
+  # but only for the master branch
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  run-linters:
+    name: Run linters
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out Git repository
+        uses: actions/checkout@v2
+
+      - name: Set up Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 16
+
+      # ESLint and Prettier must be in `package.json`
+      - name: Install Node.js dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run linters
+        uses: wearerequired/lint-action@v2
+        with:
+          eslint: true
+          prettier: true
```

Comment:
You could try:

```yaml
        with:
          eslint: false
          prettier: true
          prettier_auto_fix: false
```

If that doesn't work, perhaps try using the `command_prefix` to echo the commands being executed:

```yaml
        with:
          eslint: false
          prettier: true
          prettier_auto_fix: false
          prettier_command_prefix: echo
```

Until you get some actual output from the action it's going to be hard to debug this!

Reply:
Thanks for the help! We were able to understand things better after this :)